src/Database/Type/DateTimeType.php

```diff
@@ -119,9 +127,32 @@ public function toDatabase($value, Driver $driver)
 
         $format = (array)$this->_format;
 
+        if ($this->dbTimezone !== null
+            && $this->dbTimezone !== $value->getTimezone()->getName()
+        ) {
+            $value = $value->setTimezone(new DateTimeZone($this->dbTimezone));
```

Comment:
I am wondering if instead of creating a `DatetimeZone` instance here on each call, it would be more efficient to set `$_dbTimezone` property itself to `DatetimeZone` instance instead of string.

Reply:
We could create a single timezone instance when the timezone is first setup. That would save multiple objects from being created.